repositories/rcl.BUILD.bazel

```diff
@@ -55,3 +56,24 @@ ros2_c_library(
         "@ros2_rosidl//:rosidl_runtime_c",
     ],
 )
+
+ros2_c_library(
```

Comment:
AFAICS, this is a new library not strictly necessary for migration to humble. WDYT?

Reply:
Removed